export/build

```diff
@@ -77,13 +77,35 @@ def highlight(text):
     return "".join(l)
 
 
+def make_header(header):
+    def texify(text):
+        return text.replace("$O", "$\\O").replace("*", "\cdot ")
+
+    keywords = ["Time", "Memory", "Description", "Usage"]
+
+    l = []
+    for h in keywords:
+        if h in header:
+            if h == "Usage":
+                # TODO: quite a hack
+                l.append("{\\leftskip=\\usagedimen\\item{\\bf Usage:}\\simpleverb|%s\\egroup\\par}" % highlight(header[h]))
```

Comment:
A raw string would improve readability here, allowing you to write `\` and not `\\`
```
r"\leftskip=\usage..."
```

Reply:
Why is "usage" handled separately and is so different? (I see - it's a code snippet. Would be worth mentioning somewhere)